examples/avatar/agent_worker.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import logging
+import sys
+from dataclasses import asdict, dataclass
+from functools import partial
+
+import httpx
+from dotenv import load_dotenv
+from livekit import api
+from livekit.agents import JobContext, WorkerOptions, WorkerType, cli
+from livekit.agents.pipeline import AgentTask, PipelineAgent
+from livekit.agents.pipeline.datastream_io import DataStreamOutput
+from livekit.agents.pipeline.io import PlaybackFinishedEvent
+from livekit.agents.pipeline.room_io import RoomInput, RoomInputOptions
+from livekit.plugins import openai
+
+logger = logging.getLogger("avatar-example")
+logger.setLevel(logging.INFO)
+
+load_dotenv()
+
+
+AVATAR_IDENTITY = "avatar_worker"
+
+
+@dataclass
+class AvatarConnectionInfo:
+    room_name: str
+    url: str  # LiveKit server URL
+    token: str  # Token for avatar worker to join
+
+
+async def launch_avatar_worker(
+    ctx: JobContext, avatar_dispatcher_url: str, avatar_identity: str
+) -> None:
+    """Wait for worker participant to join and start streaming"""
+    # create a token for the avatar worker
+    # TODO(long): do we need to set agent=True here? in playground if not the video track is not automatically displayed
```

Comment:
we might want a new kind to distinguish but we probably shouldn't join with the standard participant kind